pkgs/development/python-modules/deeplabcut/default.nix

```diff
@@ -0,0 +1,136 @@
+{ lib, buildPythonPackage, fetchPypi
+, cudaSupport ? false
+, python3
+, fetchzip
+, tensorflow
+, setuptools_scm
+, curl
+, gzip
+, gnutar
+, certifi
+, chardet
+, click
+, easydict
+, h5py
+, imageio
+, ipython
+, matplotlib
+, moviepy
+, numpy
+, nvidia_x11 ? null
+, cudatoolkit ? null
+, cudnn ? null
+, opencv3
+, pandas
+, patsy
+, python
+, pyyaml
+, requests
+, ruamel_yaml
+, setuptools
+, scikitimage
+, scikitlearn
+, scipy
+, six
+, statsmodels
+, tables
+, tqdm
+, tkinter
+, wheel
+, wxPython_4_0
+}:
+
+buildPythonPackage rec {
+  pname = "deeplabcut";
+  version = "2.0.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "19gxd1nij39lbvlpsmx0ibnfcadxbk5n5a3qsyrgv68r6k6i8rk8";
+  };
+
+  resnet50 = fetchzip {
+    url = "http://download.tensorflow.org/models/resnet_v1_50_2016_08_28.tar.gz";
+    sha256 = "0n9k24ipgivvvd42h2bn26n4yjab0103ikjwhgcg668vbd5mv587";
+ };
+  resnet101 = fetchzip {
+    url = "http://download.tensorflow.org/models/resnet_v1_101_2016_08_28.tar.gz";
+    sha256 = "0h7b963px3xma936zysj54c8l930h3jyss7jv0gzbxalvxzq459x";
+ };
+
+  nativeBuildInputs = [ setuptools_scm curl gzip gnutar];
```

Comment:
What do you need `curl` for? This won't be able to download anything from the internet in our sandboxes, the only exception is fixed-output derivations.

Reply:
I don't!